### PR TITLE
Adjust D24_UNORM_S8_UINT description

### PR DESCRIPTION
**Pull request by lexaknyazev**

### CODE REVIEW

File: ktxspec.adoc [hunk 1]
```diff
@@ -470,6 +470,12 @@ value in the eight LSBs and zeros in the eight MSBs.
 
 `VK_FORMAT_D24_UNORM_S8_UINT` is defined as one 32-bit word per texel
 with the S8 value in the eight LSBs of the word and the D24 value in the MSBs.
+[TIP]
+====
+This layout matches OpenGL's `GL_UNSIGNED_INT_24_8` type. Uploading such data
+via other GPU APIs, such as Direct3D 11 or Metal, usually requires swapping the
+components, i.e., performing right rotation by 8 bits.
```
Comment (MarkCallow) [inline ktxspec.adoc:477]: Please add info about Vulkan loading to this tip.

**Comment by MarkCallow** [comment]
Do you want to publish an update containing this?

File: ktxspec.adoc (hunk 1 above)
Comment (lexaknyazev) [reply ktxspec.adoc:477]: Vulkan is covered above as it requires separate uploads anyway.

**Comment by lexaknyazev** [comment]
> Do you want to publish an update containing this?

Since it's just a non-normative tip, let's accumulate more changes.

**Pull request merged**